sdk/evaluation/azure-ai-evaluation/samples/evaluation_samples_dsb_evaluation.py

```diff
@@ -0,0 +1,43 @@
+import asyncio
+from dotenv import load_dotenv
+import os
+from azure.ai.evaluation.simulator import AdversarialScenario
+from azure.identity import DefaultAzureCredential
+from azure.ai.evaluation import DSBEvaluation, DSBEvaluator
+from typing import Optional
+
+def test_target_context(query: str) -> tuple:
+    return "Mock response", "Mock context"
+
+def test_target(query: str) -> str:
+    return "Mock response"
+
+if __name__ == "__main__":
+    load_dotenv()
+
+    azure_ai_project = {
+        "subscription_id": os.environ.get("AZURE_SUBSCRIPTION_ID"),
+        "resource_group_name": os.environ.get("AZURE_RESOURCE_GROUP_NAME"),
+        "project_name": os.environ.get("AZURE_PROJECT_NAME"),
+    }
+
+    model_config = {
+        "azure_endpoint": os.environ.get("AZURE_OPENAI_ENDPOINT"),
+        "azure_deployment": os.environ.get("AZURE_OPENAI_DEPLOYMENT"),
+    }
+
+    source_text = "Mock source text"
+
+    credential = DefaultAzureCredential()
+
+    dsb_evaluation = DSBEvaluation(model_config=model_config, azure_ai_project=azure_ai_project, credential=credential)
```

Comment:
```suggestion
    dsb_evaluation_content_safety = DSBEvaluation(model_config=model_config, azure_ai_project=azure_ai_project, credential=credential)
```

Reply:
Moved example of usage to PR description, will demonstrate this behavior there